src/mainwindow.cpp

```diff
@@ -1149,9 +1167,9 @@ void GeQueryInfoViz(TreeModel* model)
                     {
                         // Hyper does not use square brackets for table names, but the table names
                         // we log in sql-temp-table events have them.
-                        tableName = "[" + tableName + "]";
+                        tableName = "[" % tableName % "]";
                     }
-                    outputFedTempTable += queryHash + ",\"" + tableName + "\"\n";
+                    outputFedTempTable += queryHash % ",\"" % tableName % "\"\n";
```

Comment:
Is there a reason to concatenate with % instead of +? 
Also, if we use %, should we use %= instead of += to be consistent?